src/components/file/migrations/ChangeFileToFileRels.migration.ts

```diff
@@ -0,0 +1,44 @@
+import { node, relation } from 'cypher-query-builder';
+import { BaseMigration, Migration } from '../../../core';
+
+@Migration('2022-04-25T14:46:26')
+export class ChangeFileToFileRels extends BaseMigration {
+  async up() {
+    // first, invert the relationship from incoming to outgoing
+    const res = await this.db
+      .query()
+      .match([
+        node('parent', 'FileNode'),
+        relation('in', 'rel', 'parent'),
+        node('child'),
+      ])
+      .raw(`CALL apoc.refactor.invert(rel) yield input, output`)
+      .return<{ input: number }>('input')
+      .first();
+    this.logger.info(
+      `${res?.input ?? 0} FileNodes' parent relationships inverted`
+    );
+
+    // next, rename 'parent' to 'child'
+    const relNameChangeRes = await this.db
+      .query()
+      .match([
+        node('parent', 'FileNode'),
+        relation('out', 'p', 'parent'),
+        node('child'),
+      ])
+      .raw(
+        `
+        with collect(p) AS rels
+        CALL apoc.refactor.rename.type("parent", "child", rels )
```

Comment:
Can we combine these two queries? Either one after the other or maybe both in sub queries after matching the appropriate relationships?